### PR TITLE
[Feature] Message can be edited

### PR DESCRIPTION
**Pull request by EliasBoulharts**

### CODE REVIEW

File: service_catalog/views/request.py [hunk 1]
```diff
@@ -72,6 +72,30 @@ def request_comment(request, request_id):
     return render(request, "service_catalog/common/request-comment.html", context)
 
 
+@login_required
+@permission_required_or_403('service_catalog.comment_request', (Request, 'id', 'request_id'))
+def request_comment_edit(request, request_id, comment_id):
+    request_comment = get_object_or_404(RequestMessage, id=comment_id)
+    if request.method == "POST":
+        form = RequestMessageForm(request.POST or None, request.FILES or None, sender=request.user,
```
Comment (Sispheor) [inline service_catalog/views/request.py:80]: Do not override the sender.
Need to be the same so we still understand who said what.

**Review by Sispheor — CHANGES_REQUESTED** [review]
Changes to be done before merging this one

File: service_catalog/views/request.py [hunk 2]
```diff
@@ -72,6 +72,30 @@ def request_comment(request, request_id):
     return render(request, "service_catalog/common/request-comment.html", context)
 
 
+@login_required
+@permission_required_or_403('service_catalog.comment_request', (Request, 'id', 'request_id'))
```
Comment (Sispheor) [inline service_catalog/views/request.py:76]: test to be added to check that only the owner or admin can edit the message.
Same for support.

**Review by Sispheor — CHANGES_REQUESTED** [review]
Tests to be added

File: service_catalog/views/request.py (hunk 1 above)
Comment (EliasBoulharts) [reply service_catalog/views/request.py]: done

File: service_catalog/views/request.py (hunk 2 above)
Comment (EliasBoulharts) [reply service_catalog/views/request.py:76]: done

**Comment by Sispheor** [comment]
LGTM 👍

**Pull request merged**